include/psa/crypto_sizes.h

```diff
@@ -914,13 +914,13 @@
 #define PSA_EXPORT_KEY_OUTPUT_SIZE(key_type, key_bits)                                              \
     (PSA_KEY_TYPE_IS_UNSTRUCTURED(key_type) ? PSA_BITS_TO_BYTES(key_bits) :                         \
      PSA_KEY_TYPE_IS_DH(key_type) ? PSA_BITS_TO_BYTES(key_bits) :                                   \
-     (key_type) == PSA_KEY_TYPE_RSA_KEY_PAIR ? PSA_KEY_EXPORT_RSA_KEY_PAIR_MAX_SIZE(key_bits) :     \
+     PSA_KEY_TYPE_IS_RSA_KEY_PAIR (key_type) ? PSA_KEY_EXPORT_RSA_KEY_PAIR_MAX_SIZE(key_bits) :     \
```

Comment:
This  is the only change that we maintain I think. The rest of the file is just u->U which is an FSP standard. Can you revert it back to "u" so that we are only maintaining that one change?

Reply:
If I do that then our pipeline starts to fail.

Reply:
could you post a link to that failing pipeline?

Reply:
It will start throwing error like: error: integer literal has suffix 'u', which is not uppercase [readability-uppercase-literal-suffix,-warnings-as-errors]
Link: http://pbgitap01.rea.renesas.com/peaks/peaks/-/jobs/10988537